checklistsbielik.txt

```diff
@@ -0,0 +1,20 @@
+Check list 
+1. Verify that User can click on 'Woman' section 'Categories' link, link is opened
+2. Verify that User can click on 'Specials' section 'Information', link is opened
+3. Verify that User can click on 'New products' section 'Information', link is opened
+4. Verify that User can click on 'Best sellers' section 'Information', link is opened.
+5. Verify that User can click on 'Our stores' section 'Information', link is opened.
+6. Verify that User can click on 'Contact us' section 'Information', link is opened.
+7. Verify that User can click on 'Terms and conditions of use' section 'Information', link is opened.
+8. Verify that User can click on 'About us' section 'Information', link is opened.
+9. Verify that User can click on 'Sitemap' section 'Information', link is opened.
+10. Verify that User can click on 'My orders' section 'My account', link is opened.
+11. Verify that User can click on 'My credit slips' section 'My account', link is opened.
+12. Verify that User can click on 'My addresses' section 'My account', link is opened.
+13. Verify that User can click on 'My personal info' section 'My account', link is opened.

```

Comment:
Are we interested in what links are displayed in bottom menu? Would it be worth to verify some list?